build.gradle

```diff
@@ -981,8 +982,14 @@ project('standalone') {
         if (systemProperties.get("extDirs") != null) {
             classpath += files(systemProperties.get("extDirs"))
         }
-
-        jvmArgs "-Xdebug", "-Xrunjdwp:server=y,transport=dt_socket,address=8000,suspend=n"
+        
+        def jvmDebugArgs=System.getenv("JVM_DEBUG_ARGS")
+        if(jvmDebugArgs !=null&& jvmDebugArgs !=""){
```

Comment:
Please use standard formatting for spaces

Reply:
Sure, will take care of this point